config/urls.py

```diff
@@ -1,60 +1,40 @@
-"""explorer URL Configuration
-
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/2.1/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  path('', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.urls import include, path
-    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
-"""
 from django.conf import settings
-from django.contrib import admin
 from django.urls import include, path
 
 from scan.views.accounts import AccountsListView, AddressDetailView
+from scan.views.aliases import AliasListView
 from scan.views.assets import (
     AssetDetailView,
-    AssetMintingDetailView,
     AssetDistributionDetailView,
     AssetHoldersListView,
     AssetListView,
+    AssetMintingDetailView,
     AssetTradesListView,
     AssetTransfersListView,
-    
 )
 from scan.views.ats import AtDetailView, AtListView
 from scan.views.blocks import BlockDetailView, BlockListView
+from scan.views.cashbacks import CBListView
+from scan.views.distribution import DistributionListView
+from scan.views.forged_blocks import ForgedBlocksListView
 from scan.views.index import index
+from scan.views.json import TopAccountsJson, getSNRjson, getStatejson
 from scan.views.marketplace import (
     MarketPlaceDetailView,
     MarketPlaceListView,
     MarketPlacePurchasesListView,
 )
+from scan.views.miners import MinerListView
 from scan.views.peers import (
     PeerMonitorDetailView,
     PeerMonitorListView,
     peers_charts_view,
 )
-
-from scan.views.json import (
-    getSNRjson,
-    getStatejson,
-    TopAccountsJson,
-)
-
```

Comment:
Is this not longer needed ? Did you checked that ?

Reply:
In urls.py this import not use, so it can be deleted.

Reply:
> Is this not longer needed ? Did you checked that ?

Sorry. This import has been moved to line 21. Moved to match PEP8.

Reply:
ah okay , thanks - and without () that will work on line 21 ?

Reply:
> ah okay , thanks - and without () that will work on line 21 ?

Yes, () is just a way of displaying multiple objects, where each object is on a new line.